packages/eslint-config/package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "@harmony/eslint-config",
+    "name": "@river/eslint-config",
```

Comment:
Our package names should be river-build everywhere. I will make a task to rename the rest

Reply:
okay, I renamed the stuff that I changed from @river to @river-build